cli/src/main/java/metamer/cmdparser/CliHandler.java

```diff
@@ -297,6 +335,11 @@ private static String printHelp(final Options options) {
         return stringWriter.toString();
     }
 
+    private static String showLog() throws IOException {
+        final File file = new File(tmpDir, logName);
+        return Files.lines(Paths.get(file.getAbsolutePath())).map(it -> it + "\n").collect(toList()).toString();
```

Comment:
Обрати внимание как сейчас печатается лог:
```bash
java -jar cli-1.0.0.jar -v 
```
> WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
metamer.cmdparser.exception.NoLength: No length of kmers was provided, a correct length must be provided
metamer.cmdparser.exception.NoFormat: No format of data was provided, a correct format must be provided
[2019-07-16T22:25:07,160 [INFO ] (CliHandler.java:106) - START
, 2019-07-16T22:25:07,164 [INFO ] (CliHandler.java:149) - k value = null
, 2019-07-16T22:25:07,164 [INFO ] (CliHandler.java:151) - format value = null
, 2019-07-16T22:25:07,182 [INFO ] (CliHandler.java:166) - strategy : from stdin to stdout
]

Посмотри внимательнее на контракт `Files`, там есть более подходящий для нашего сценария метод `#readString(Path)`, который сразу читает файл в виде строки. Замени, пожалуйста, тут на использование него.